src/class-parsely.php

```diff
@@ -1346,9 +1346,18 @@ public function script_loader_tag( string $tag, string $handle, string $src ): s
 				'wp-parsely-recommended-widget',
 			)
 		) ) {
-			// Have CloudFlare Rocket Loader ignore these scripts:
-			// https://support.cloudflare.com/hc/en-us/articles/200169436-How-can-I-have-Rocket-Loader-ignore-specific-JavaScripts-.
-			$tag = preg_replace( '/^<script /', '<script data-cfasync="false" ', $tag );
+			/**
+			 * Filter whether to include the CloudFlare Rocket Loader tag (`data-cfasync=false`) in the script. Only
+			 * needed if the site being served is behind Cloudflare. Should return false otherwise.
+			 * https://support.cloudflare.com/hc/en-us/articles/200169436-How-can-I-have-Rocket-Loader-ignore-specific-JavaScripts
+			 *
+			 * @since 3.0.0
+			 *
+			 * @param bool $enabled True if enabled, false if not.
+			 */
+			if ( apply_filters( 'wp_parsely_enable_cfasync_tag', false ) ) {
```

Comment:
```suggestion
			if ( apply_filters( 'wp_parsely_enable_cfasync_tag', false, $handle ) ) {
```

If we make the handle available, then it allows folks to conditionally apply this to only some of the 4 Parse.ly scripts if they wish.